pkg/apicheck/check.go

```diff
@@ -227,12 +226,13 @@ func (c *ApiConnectivityCheck) popNodes(nodes *[][]v1.NodeAddress, count int) []
 	//todo maybe we should pick nodes randomly rather than relying on the order returned from api-server
 	addresses := make([]string, count)
 	for i := 0; i < count; i++ {
-		nodeAddresses := (*nodes)[i]
-		if len(nodeAddresses) == 0 || nodeAddresses[0].Address == "" {
+		// TODO: shall we need to get "count" addresses anyway, replacing empty IP with another Node?
+		address := (*nodes)[i]
```

Comment:
Nit: maybe use something more specific than `address` (i.e for example podIp or podIpAddress)